source/core/Date.cpp

```diff
@@ -113,22 +167,21 @@ namespace Vireo {
         return currentYear;
     }
 
-#if kVireoOS_windows
-    std::string abbreviateTimeZoneName(std::string input)
+    void abbreviateTimeZone(const char * input, char output[kTempCStringLength])
     {
         char prev = ' ';
-        std::string output;
-        for (int i = 0; i < input.length(); i++)
+        int j = 0;
+        for (int i = 0; i < strlen(input); i++)
         {
             if (prev == ' ' && input[i] != ' ')
             {
-                output += input[i];
+                output[j] = input[i];
+                ++j;
             }
             prev = input[i];
         }
-        return output;
+        output[j] = 0;
```

Comment:
No. LV's behavior seems to be locale specific. I haven't spent much time on this. I am not sure whether we want to abbreviate at all just to make the tests pass or make the tests platform specific and go with whatever the default is for the platform (don't abbreviate for Windows and use the abbreviated timezones for Linux/Mac). Let's make a final decision on that in the future, we can spend a bit more effort on this to clean this up.